src/gateway/kong-manager/get-started/services-routes.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Services and Routes
```

Comment:
```suggestion
title: Configuring Services and Routes
```
Just a suggestion to distinguish this from Understanding Kong: Services or Understanding Kong: Routes. I think having the name as "Services and Routes" in the nav is okay though, just thinking of Google searches to find this page.

Reply:
added